Diagram: add Getting Started tutorial (#6863)
DevExpress/devextreme-documentation: concepts/05 UI Components/Diagram/00 Getting Started with Diagram/10 Bind Diagram to Data.md

Use the [nodes.dataSource](/api-reference/10%20UI%20Components/dxDiagram/1%20Configuration/nodes/dataSource.md '/Documentation/ApiReference/UI_Components/dxDiagram/Configuration/nodes/#dataSource') property to bind Diagram to a list of nodes. This tutorial uses a [linear array](/Documentation/Guide/UI_Components/Diagram/Data_Binding/#Linear_Array) of nodes as the data source for the initial Diagram. In this case, the required properties to specify are [nodes.keyExpr](/api-reference/10%20UI%20Components/dxDiagram/1%20Configuration/nodes/keyExpr.md '/Documentation/ApiReference/UI_Components/dxDiagram/Configuration/nodes/#keyExpr') and [nodes.parentKeyExpr](/api-reference/10%20UI%20Components/dxDiagram/1%20Configuration/nodes/parentKeyExpr.md '/Documentation/ApiReference/UI_Components/dxDiagram/Configuration/nodes/#parentKeyExpr'). The tutorial also includes [nodes.textExpr](/Documentation/ApiReference/UI_Components/dxDiagram/Configuration/nodes/#textExpr).

> To learn more about different ways of Diagram data binding, refer to the following help topic: [Data Binding](/Documentation/Guide/UI_Components/Diagram/Data_Binding/).

Reviewer: ```suggestion
For details about different ways to bind Diagram data, refer to the following help topic: [Data Binding](/Documentation/Guide/UI_Components/Diagram/Data_Binding/).
```
